phpunit.xml

```diff
@@ -7,7 +7,9 @@
          convertNoticesToExceptions="true"
          convertWarningsToExceptions="true"
          processIsolation="false"
-         stopOnFailure="false">
+         stopOnFailure="false"
+         verbose="true"
```

Comment:
verbose is included in the github workflow.
Do you need to add it here?

Reply:
Fair point. Thanks.